website/pages/en/index.js

```diff
@@ -164,7 +164,7 @@ const Model = props => (
   <Block id="model">
     {[
       {
-        content: '<div class="typeset">The template model defines the structure of data that are used in the natural language and logic. Accord Project templates use the <a href="https://github.com/hyperledger/composer-concerto">Composer Concerto Modeling Language</a>.</div>',
+          content: '<div class="typeset">The template model defines the structure of data that are used in the natural language and logic. Accord Project templates use the <a href="https://github.com/hyperledger/composer-concerto">Composer Concerto Modeling Language</a>.</div>',
```

Comment:
I think this link still needs to be changed.

Reply:
well
If I remove this sentence, than, I think not :D

Reply:
AH! Yes, I think it's still early for me, I was a bit confused.

Reply:
@mttrbrts I think just still waiting on confirmation on the removal of the second sentence.

Reply:
I'm confused by this thread, I don't see a sentence removed and it still uses the old link and name for Concerto.

Reply:
@jeromesimeon the main question was : should it be removed completely, or not? I revert all changes back before getting the final requirements, about the name - yes, that will be done